src/status_im/utils/universal_links/core.cljs

```diff
@@ -81,8 +86,8 @@
       (navigation/navigate-to-cofx (assoc-in cofx [:db :contacts/identity] public-key) :profile nil))))
 
 (fx/defn handle-eip681 [cofx url]
-  {:dispatch-n [[:navigate-to :wallet]
-                [:wallet/fill-request-from-url url]]})
+  (choose-recipient/resolve-ens-addresses url :deep-link)
+  {:dispatch [:navigate-to :wallet-send-transaction]})
```

Comment:
this won't work you need to use `fx/merge `

```clojure
(fx/merge cofx
                  (choose-recipient/resolve-ens-addresses url :deep-link)
                   (navigation/navigate-to-cofx  :wallet nil))
```
also notice that we navigate to wallet in develop

Reply:
Fixed in new commit.